auth/src/main/java/org/aerogear/mobile/security/SecurityService.java

```diff
@@ -83,19 +105,19 @@ public SecurityCheckResult check(@NonNull final SecurityCheck securityCheck) {
      * Perform a single {@link SecurityCheckType} , get the {@link SecurityCheckResult result} and
      * publish a {@link SecurityCheckResultMetric} based on the result.
      *
-     * @param securityCheckType The type of check to execute
-     * @param metricsService The metrics service to use
+     * @param securityCheckType The {@link SecurityCheckType} to execute
+     * @param metricsService {@link MetricsService}
      * @return {@link SecurityCheckResult}
      */
     public SecurityCheckResult checkAndSendMetric(final SecurityCheckType securityCheckType, final MetricsService metricsService) {
         return checkAndSendMetric(securityCheckType.getSecurityCheck(), metricsService);
```

Comment:
I think it doesn't make sense to allow `null` for the metricService in the `checkAndSendMetric` method.
WDYT?

Reply:
Ya that makes sense, I'll refactor this